clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10878,4 +10878,6 @@ def err_ext_int_bad_size : Error<"%select{signed|unsigned}0 _ExtInt must "
                                  "have a bit size of at least %select{2|1}0">;
 def err_ext_int_max_size : Error<"%select{signed|unsigned}0 _ExtInt of bit "
                                  "sizes greater than %1 not supported">;
+def err_aux_target_builtin_in_device_code : Error<
+  "AUX target specific builtins should not be present in device code">;                                 
```

Comment:
The users don't know anything about AUX-target, that is implementation specific.  Word this having to do with host/device.

Phrase this as something like:
"host specific builtin cannot be used in device functions"

Someone else mind wordsmithing?